src/Controller/CarController.php

```diff
@@ -171,30 +172,40 @@ public function show(#[ValueResolver(CarValueResolver::class)]
             new OA\Response(response: 400, description: 'Invalid input data')
         ]
     )]
-        public function new(Request $request,EventDispatcherInterface $eventDispatcher): Response
+    public function new(Request $request, EventDispatcherInterface $eventDispatcher): Response
     {
-        $car = new Car();
         $user = $this->security->getUser();
 
         if (!$user) {
             return $this->redirectToRoute('app_login');
         }
 
-        $car->setUser($user);
```

Comment:
so you don't set user anymore, how the code works? How it knows who is the user, why is this removed?